src/frameworks/index.ts

```diff
@@ -411,7 +411,12 @@ You can link a Web app to a Hosting site here https://console.firebase.google.co
         },
       ]);
 
-      if (!targetNames.includes("functions")) targetNames.unshift("functions");
+      if (!targetNames.includes("functions")) {
+        targetNames.unshift("functions");
+        if (options.only) {
+          options.only = `${options.only},functions:firebase-frameworks-${site}`;
```

Comment:
Can we defensively DRY up the codebase name? Incase we need to add a transformation or something down the road.